doc/release/1.17.0-notes.rst

```diff
@@ -18,6 +18,13 @@ New functions
 Deprecations
 ============
 
+Deprecate `numpy.dist_utils.exec_command` and `numpy.dist_utils.temp_file_name`
```

Comment:
These need double backticks (or they'll turn into broken links after we remove the functions).

And `dist_utils` is spelled `distutils`

Reply:
fixed